### PR TITLE
Few suggested fixes to the 57th edition

### PR DESCRIPTION
**Pull request by sivaraam**

### CODE REVIEW

File: rev_news/drafts/edition-57.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ This edition covers what happened during the month of October 2019.
   `git diff` that would allow apps like `tig` and `magit` to show
   moved lines.
 
-  I'd also like to improve git push and pull to extend the idea of
+  I'd also like to improve Git push and pull to extend the idea of
```
Comment (chriscool) [inline rev_news/drafts/edition-57.md]: I think here it's ok to use "git". Perhaps we should even have back-ticks around "git push" and around "git pull" too.

**Comment by chriscool** [comment]
Except for the comment I left, everything looks good! Thanks!

File: rev_news/drafts/edition-57.md (hunk 1 above)
Comment (sivaraam) [reply rev_news/drafts/edition-57.md]: I was on the fence with that particular instance. I thought it best to change it and let some else point out if its wrong 😉

**Comment by chriscool** [comment]
Thanks! Merged!

**Pull request merged**